packages/core/cli/package.json

```diff
@@ -16,12 +16,14 @@
   "dependencies": {
     "inquirer": "^7.3.3",
     "chokidar": "^3.3.1",
-    "consola": "^2.10.1",
-    "@vue-storefront/nuxt-theme": "^0.0.3"
+    "consola": "^2.10.1"
   },
   "devDependencies": {
     "@vue-storefront/commercetools-theme": "^0.1.0"
   },
+  "peerDependencies": {
```

Comment:
Are you sure this will work as a peerDependency? I guess it probably won't

Reply:
It is dependency of commercetools-theme only in development mode. If someone installs CLI globally - then he/she won't have commercetools-theme - only generated template. That's why he/she won't have nuxt-theme - which is needed in CLI (there are imports from nuxt-theme) - and it will throw an error all the time

Reply:
Good catch, I reverted this change